src/server.js

```diff
@@ -25,14 +22,33 @@ app.post('/rasterize', async (req, res) => {
   const camera = initCamera(45, aspect, -50, 40, 120, 0)
   initLights(scene)
 
-  const model = await loadIfcUrl(req.body.url)
+  let ifcURL = req.body.url
+  let coordinates = []
+
+  const url = new URL(ifcURL)
+  if (url.hostname === 'bldrs.ai') {
+    const b = parseURLFromBLDRS(url)
+    ifcURL = b.target.url
+
+    if ('c' in b.params) {
+      coordinates = b.params['c'].split(',').map(f => parseFloat(f))
+    }
+  }
+
+  const model = await loadIfcUrl(ifcURL)
   model.position.set(-40, 0, 0)
```

Comment:
I think this client may be out of date?  The model position shouldn't be hardcoded

Reply:
That's the code that's in the server implementation.

I believe your earlier PR change where you introduced the camera changes only targeted the command line client.

But I'll remove this from the server implementation.